.github/workflows/translation-advice.yml

```diff
@@ -7,7 +7,9 @@ on:
     paths:
       - "!**"
       - "locale/**"
+      - "!locale/en/**"
       - "web/src/locales/**"
+      - "!web/src/locales/en.ts"
```

Comment:
Do we need to add pseudo-LOCALE.ts here as well?

Reply:
All of the web/src locales are generated at build time: https://github.com/goauthentik/authentik/tree/main/web/src/locales

Reply:
Ah, so we can entirely remove the web stuff in here.

For my own comprehension, does the build download them from transifex then?

Reply:
Nothing is downloaded, the translation source files are in GitHub and get converted to typescript during build, anytime anything changes in transifex it gets PR'd to the repo